### PR TITLE
[Merged by Bors] - chore: tidy various files

### PR DESCRIPTION
**Pull request by Ruben-VandeVelde**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Geometry/Manifold/PartitionOfUnity.lean [hunk 1]
```diff
@@ -503,7 +503,8 @@ theorem exists_isSubordinate {s : Set M} (hs : IsClosed s) (U : ι → Set M) (h
   haveI : LocallyCompactSpace H := I.locally_compact
   haveI : LocallyCompactSpace M := ChartedSpace.locallyCompact H M
   haveI : NormalSpace M := normal_of_paracompact_t2
-  -- porting note: split `rcases` into `have` + `rcases`
+  -- porting note(https://github.com/leanprover-community/mathlib4/issues/5732):
```
Comment (alexjbest) [inline Mathlib/Geometry/Manifold/PartitionOfUnity.lean:506]:
```suggestion
  -- porting note(https://github.com/leanprover/std4/issues/116):
```
seeing as the mathlib one was closed in favour of that

**Comment by alexjbest** [comment]
LGTM

**Comment by alexjbest** [comment]
maintainer merge

File: Mathlib/Analysis/InnerProductSpace/Adjoint.lean [hunk 2]
```diff
@@ -40,6 +40,7 @@ adjoint
 
 -/
 
+local macro_rules | `($x ^ $y)   => `(HPow.hPow $x $y) -- Porting note: See issue #2220
```
Comment (eric-wieser) [inline Mathlib/Analysis/InnerProductSpace/Adjoint.lean:43]: This isn't #2220; what does this refer to?

Comment (alexjbest) [reply Mathlib/Analysis/InnerProductSpace/Adjoint.lean:43]: https://github.com/leanprover/lean4/issues/2220

Comment (eric-wieser) [reply Mathlib/Analysis/InnerProductSpace/Adjoint.lean:43]:
```suggestion
local macro_rules | `($x ^ $y)   => `(HPow.hPow $x $y) -- Porting note: See issue lean4#2220
```

File: Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean [hunk 3]
```diff
@@ -54,7 +54,7 @@ As specific examples of models with corners, we define (in the file `real_instan
 With these definitions at hand, to invoke an `n`-dimensional real manifold without boundary,
 one could use
 
-  `variables {n : ℕ} {M : Type*} [TopologicalSpace M] [ChartedSpace (EuclideanSpace (Fin n)) M]
+  `variables {n : ℕ} {M : Type _} [TopologicalSpace M] [ChartedSpace (EuclideanSpace (Fin n)) M]
```
Comment (eric-wieser) [inline Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean:57]: This is a regression from merging master

**Comment by eric-wieser** [comment]
bors d+

Please fix the `Type*`/`Type _` issues before merging.

**Pull request closed without merging**